### ISSUE TITLE
Cloudflare bug

### ISSUE THREAD

**Issue by Stage4000** [cdgco/HestiaWebInterface#13]
Cloudflare plugin on hwi calls for hwi folder within hestia, however the installer still installs the cloudflare page in vwi folder (https://github.com/cdgco/HestiaWebInterface/tree/master/install/hestia/web/vwi). Change vwi directory to hwi and it works as intended (as far as I can tell).

**Comment by Stage4000** [comment]
Further information on this can be found in my previous (closed) issue comments.

**Comment by Stage4000** [comment]
https://github.com/cdgco/HestiaWebInterface/tree/master/install/hestia/web/list/rrd vwi.php needs renamed to hwi.php. calls for it are failing due to improper name just like cloudflare.

**Comment by Stage4000** [comment]
for some reason the if clause that checks referrer is always returning false, I simply removed the referrer check after changing the file name accordingly and it works.

**Comment by cdgco** [comment]
Sorry, I've been busy and haven't had a chance to look over this. To be honest, the Cloudflare integration is not great, it's pretty fragile and was one of the first integrations I built, before I built a proper plugin system. It's probably not worth making it a standalone plugin now as it's too deeply integrated in the panel, but I'm sure there are other bugs that need to be fixed. I'll accept the changes as a temporary fix, but I'd like to get things properly changed over from VWI to HWI and do some deeper testing of the Cloudflare integration.

**Comment by Stage4000** [comment]
Yeah I've still not got cloudflare to work honestly. Not exactly sure what the purpose of the plugin is I've I'm to be honest because cloudflare requires you use their nameservers as SOA.

**Comment by cdgco** [comment]
It uses the HestiaCP DNS to forward the nameservers to Cloudflare and allows you to change SSL, security level, proxy, etc, directly from the HWI panel.

**Comment by cdgco** [comment]
It requires that you have nameserver records and DNS setup correctly for your HestiaCP server first.

**Comment by Stage4000** [comment]
Well either I have something set up wrong or it doesn't work. I have my domain stage4000 using glue to point ns1 and ns2 to itself and then the cf plugin tries adding it to my panel but the record never propogate and cf is never happy about my nameservers.

**Comment by Stage4000** [comment]
Do you have Discord?
